package.json

```diff
@@ -37,16 +37,23 @@
     "safe-json-stringify": "^1.2.0"
   },
   "devDependencies": {
-    "eslint": "^7.32.0",
-    "eslint-config-airbnb": "^18.2.1",
-    "eslint-config-scality": "scality/Guidelines#71a059ad",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jsx-a11y": "^6.4.1",
-    "eslint-plugin-react": "^7.26.0",
-    "eslint-plugin-react-hooks": "^4.2.0",
-    "jsdoc": "^3.4.3",
-    "markdownlint-cli": "^0.27.1",
-    "mocha": "^8.4.0",
+    "@babel/eslint-parser": "^7.25.1",
+    "eslint": "^8.57.0",
```

Comment:
eslint 9 ?

Reply:
We cannot, it requires node 20, but then we cannot use it in our projects because they're not yet upgraded to node 20, which would make this dependency not usable... Do we want to upgrade to eslintv9 now?

```
error @eslint/config-array@0.18.0: The engine "node" is incompatible with this module. Expected version "^18.18.0 || ^20.9.0 || >=21.1.0". Got "16.20.2"
error Found incompatible module.
```

Reply:
Now it uses eslint v9, I also removed almost all devdeps, are they are unused, we should only rely on the scality eslint configuration... This will simplify things and then we can really enforce a coding style everywhere.